### ISSUE TITLE
Feature request: Add more event overrides

### ISSUE THREAD

**Issue by Wirox** [octgn/OCTGN#1810]
Currently there are only four event overrides, which doesn't offer much customization in this matter. On the other hand, there are plenty of event triggers, and many of them would also be useful in form of an event override.

Examples include:
– [`OnCounterChanged`](https://github.com/octgn/OCTGN/wiki/OCTGN-Python-3.1.0.2-API-Reference#oncounterchanged)
– [`OnTurnPaused`](https://github.com/octgn/OCTGN/wiki/OCTGN-Python-3.1.0.2-API-Reference#onturnpassed)
– [`OnMarkerChanged`](https://github.com/octgn/OCTGN/wiki/OCTGN-Python-3.1.0.2-API-Reference#onmarkerchanged)
– [`OnCardControllerChanged`](https://github.com/octgn/OCTGN/wiki/OCTGN-Python-3.1.0.2-API-Reference#oncardcontrollerchanged)

**Comment by kellyelton** [comment]
Is the purpose of this to selectively fire the event?

So in the case of `OnCardControllerChanged`, you would conditionally change the card controller?

I'm not sure what else you would override that event with?

**Comment by Wirox** [comment]
Yeah, the basic use of these overrides would be to allow or disallow certain things based on some conditions.

**Comment by Wirox** [comment]
Other event overrides that could be useful include all default OCTGN card/group actions, for example 'Bring to front' for cards, 'Look at (...)' for piles, and so on.
